### ISSUE TITLE
keyword argument `dcor_site_config_dir` is not working as expected

### ISSUE THREAD

**Issue by RaghavaAlajangi** [DCOR-dev/dcor_control#20]
The custom keyword argument `dcor_site_config_dir` is not working as expected.

Case:1 - passing custom configuration dir
```
# Command
dcor inspect --assume-yes --dcor-site-config-dir "site_dcor-docker"

# Result
Error: Invalid value for '--dcor-site-config-dir': File 'site_dcor-docker' is a directory.
```
Case:2 - passing configuration file path
```
# Command
dcor inspect --assume-yes --dcor-site-config-dir "site_dcor-docker/dcor_config.json"

# Result
NotADirectoryError: [Errno 20] Not a directory: '/srv/app/site_dcor-docker/dcor_config.json/dcor_config.json'
```

Case:3 - defining configuration dir as a `env` variable
```
# Set env variable
DCOR_SITE_CONFIG_DIR=/srv/app//site_dcor-docker

# Result
File "/srv/app/.local/lib/python3.10/site-packages/dcor_control/inspect/config_ckan.py", line 254, in is_site_config_dir_applicable
    cfg = json.loads((dcor_site_config_dir / "dcor_config.json").read_text())
TypeError: unsupported operand type(s) for /: 'str' and 'str'
```

**Comment by paulmueller** [comment]
Thanks!

- Case 1 is a bug
- Case 2 should not work (because a directory is required)
- Case 3 is a bug (note that the directory `/srv/app//site_dcor-docker` might be invalid due to the double `//`)

**Comment by paulmueller** [comment]
fixed in version 0.11.8
